vyper/ast/grammar.lark

```diff
@@ -283,8 +271,22 @@ _IN: "in"
        | "+"  power -> uadd
        | "-"  power -> usub
        | "~"  power -> invert
-?power: atom
-      | power _POW  atom -> pow
+
+// TODO: add factor rule
+?power: external_call
```

Comment:
this leg of the rule is weird, I know it was like that but why :thinking:

Reply:
I don't really know. It's out of line with the normal precedence table for factors.

Overall I'm not a huge fan of having to maintain our own grammar tbh.